code/obj/item/device/pda2/smallprogs.dm

```diff
@@ -559,10 +559,12 @@ Code:
 	var/obj/machinery/power/pt_laser/laser
 	var/obj/machinery/power/generatorTemp/generator
 	var/obj/machinery/carouselpower/carousel
+	var/obj/machinery/atmospherics/binary/reactor_turbine/nuke_turbine
+	var/obj/machinery/atmospherics/binary/nuclear_reactor/nuke_reactor
 
 	proc/find_machinery(obj/ref, type)
 		if(!ref || ref.disposed)
-			ref = locate(type) in machine_registry[MACHINES_POWER]
+			ref = locate(type) in machine_registry[MACHINES_POWER || MACHINES_FISSION]
```

Comment:
this does not do what you think it does, `3 || 4 == 3`

Reply:
`locate(...) in (machine_registry[power] + machine_registry[fission])` should work (pseudocode)

Reply:
I see
I have no idea how the machine registry and the defines actually work, see, so i didnt know how to fix it. I'll try this

Reply:
update: this has not worked
Either `locate()` doesn't play nice with the `+`, or the things are not in `MACHINES_FISSION`

Reply:
update: reactors and turbines weren't in `MACHINES_FISSION`. Adding that in fixed the whole thing.